src/getTime.js

```diff
@@ -18,7 +18,8 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const time = str.match(/\b(?:[0-1][0-9]|[0-2][0-3]):[0-5][0-9]\b/);
```

Comment:
можна коротше трошки
```suggestion
  const time = str.match(/\b(?:[01]\d|2[0-3]):[0-5]\d\b/);
```

Reply:
дякую, прийняла до уваги!